xhtml2pdf/xhtml2pdf_reportlab.py

```diff
@@ -32,10 +32,9 @@
 try:
     import StringIO
 except Exception:
-    from io import StringIO
-    StringIO_old = StringIO
+    from io import BytesIO
     class StringIO(object):
-        StringIO = StringIO_old
+        StringIO = BytesIO
```

Comment:
Nope, 2.5 is of the past. If people wanna use that, they should also use old versions of xhtml2pdf :)